src/posts/xwrf/index.md

```diff
@@ -0,0 +1,162 @@
+---
+title: 'Making WRF output data pythonic using xWRF'
+date: '2022-09-20'
+authors:
+  - name: Jon Thielen
+    github: jthielen
+  - name: Lukas Pilz
+    github: lpilz
+---
+
+![xWRF logo](https://github.com/xarray-contrib/xwrf/blob/main/docs/source/_static/xwrf_logo_bg_light.svg)
+
+In the past, when analyzing [Weather Research and Forecasting (`WRF`)](https://www.mmm.ucar.edu/weather-research-and-forecasting-model) data with python you would be restricted to using the [`wrf-python`](https://wrf-python.readthedocs.io/en/latest/) package. This is not very convenient, especially when working on very large datasets since it uses compiled Fortran code in the background essentially preventing the user from scaling to cluster-scale using utilities like [`dask`](https://www.dask.org/).
+
+Here, we present to you [`xWRF`](https://github.com/xarray-contrib/xwrf/), a lightweight interface for working with the model output in [Xarray](https://docs.xarray.dev/en/stable/). Using `xWRF` enables a seamless integration of the unique `WRF` data format into Xarray and the [Pangeo](https://pangeo.io/) software stack. It achieves this by:
```

Comment:
Feel free to mention how far this has come since the prototype I wrote about last fall :)